include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -416,6 +416,49 @@ class argument_parser
         return *sub_parser;
     }
 
+    /*!\brief Checks whether the option distinguished by `id` was set on the command line by the user.
+     * \param[in] id The option identifier to search for (must not contain dashes!).
+     * \returns `true` if option identifier `id` was set on the command line by the user.
+     */
```

Comment:
The documentation is part of the public API and should be a little bit more verbose. For example it is not clear that the function might throw. Second the documentation says that id should not have dashes. But the overloaded function would get the same information, while it is not true in this case.